### DISCUSSION TITLE
Does Waybar no longer supports the "format-alt"  after the update?

### DISCUSSION THREAD

**Discussion by DreamMaoMao** [Alexays/Waybar#2410] [General]
Whether in the network module or clock module, the mouse click cannot switch to another format display properly. There is no example in the wiki anymore. I have seen many people raising issues in the discussions, and they are all caused by the fact that "format-alt" is not properly supported anymore.

**Comment by Alexays** [comment]
Just reverted the PR that is faulty.
https://github.com/Alexays/Waybar/commit/b5ea14c896f3659b333cc474c8a491672cbc6465
